app/core/Tracker.coffee

```diff
@@ -136,6 +136,12 @@ module.exports = class Tracker extends CocoClass
         eventAction: action
       gaFieldObject.eventLabel = properties.label if properties.label?
       gaFieldObject.eventValue = properties.value if properties.value?
+
+      # Add label for tracking home page A/B testing results on GA
+      # TODO: what if label already exists, as of now there is no label for the events for which trackABResult is true.
```

Comment:
I don't think I fully understand this comment. Is the concern that the `eventLabel` duplicates an existing eventLabel?

Reply:
Yeah, i.e. what if we are already tracking some label for an event, need to figure out a way to support multiple labels or something like that.

Reply:
Could we query the db if we were concerned? Or just make the label so specific that the likelihood is very unlikely?

Reply:
I am adding this label for a/b testing group, only for google analytics(which is not stored in db), which has 3 parameters mainly: event action, event category, and event label. So I am adding the test group (a/b/c) as the label for the specific events on GA so that we can see the results for them. As of now, there was no other label for those events so I could easily add this label. The comment here means that if in future, we track a/b results for events that already have a label associated with them, then we need to figure out how to handle multiple labels in GA. Not a big concern right now since we are just running 1 a/b test for next 2 weeks, and may or may not run more after that.

Probably, I can change it to an object, for now, eg: `{testGroup: A}`, and then if there will be more labels in future we can add to this object as new keys. Will also update the comment to make it simpler.